app/src/main/java/com/sdp/movemeet/backend/providers/BackendInstanceProvider.java

```diff
@@ -0,0 +1,40 @@
+package com.sdp.movemeet.backend.providers;
+
+import androidx.annotation.VisibleForTesting;
+
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.storage.FirebaseStorage;
+
+public class BackendInstanceProvider {
+
+    @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
+    public static final BackendInstanceProvider instance = new BackendInstanceProvider();
+
+    @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
+    public static FirebaseFirestore firestore;
+    @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
+    public static FirebaseStorage storage;
+    @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
+    public static FirebaseDatabase database;
+
+    private BackendInstanceProvider() {
```

Comment:
Is this private constructor really needed ? Can't we simply define the instances directly in the attributes ?

Reply:
As before, it was imported in a merge with the Kepler branch, so I think we can talk about that in our meeting. It's maybe useful for some kind of abstraction in some tests.